fe/fe-core/src/main/java/com/starrocks/authentication/AuthenticationManager.java

```diff
@@ -44,20 +45,24 @@
 import java.net.URL;
 import java.net.URLClassLoader;
 import java.nio.charset.StandardCharsets;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
-import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.TreeMap;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 
+import static com.starrocks.mysql.privilege.AuthPlugin.AUTHENTICATION_LDAP_SIMPLE_FOR_EXTERNAL;
```

Comment:
import static is not our common code style

Reply:
i think it's fine, we have a lot usage of it in current code base.

Reply:
after some thoughts, i am gonna to replace it :-)